examples/next-13/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "example-next-13",
```

Comment:
Yeah... even weirder that the player doesn't even have it's own package. But idk, you know better how these next projects should be organized anyway.